source/binary.cpp

```diff
@@ -414,7 +431,7 @@ spv_result_t Parser::parseInstruction() {
   return SPV_SUCCESS;
 }
 
-spv_result_t Parser::parseOperand(size_t inst_offset,
+spv_result_t Parser::parseOperand(const size_t inst_offset,
```

Comment:
Not necessary.  I thought it was useful documentation for the implmemetation that the variable would not change.
Will remove, as it appears jarring to you.